build/bin/dev

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+var path = require('path');
+var sh = require('shelljs');
+var nodeCli = require('shelljs-nodecli');
+var budo = require('budo');
+var minimist = require('minimist');
+var pkg = require('../../package.json');
+var version = pkg.version.split('.');
+var argv = minimist(process.argv, {
+  boolean: ['l', 'o'],
+  alias: {
+    l: 'live',
+    o: 'open'
+  },
+  default: {
+    o: false,
+    l: false
+  }
+});
+
+version = {
+  full: pkg.version,
+  major: version[0],
+  minor: version[1],
+  patch: version[2],
+  majorMinor: version[0] + version[1]
+};
+
+var b = budo(process.cwd() + '/src/js/video.js', {
```

Comment:
these paths and what not should be fiddled with before merging. Need to figure out exactly what I can put down. I used to have `'./'` here but it's broken when running through `npm start`.